presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperatorFactory.java

```diff
@@ -123,6 +125,7 @@ private LookupJoinOperatorFactory(LookupJoinOperatorFactory other)
         referenceCount = other.referenceCount;
         outerOperatorFactory = other.outerOperatorFactory;
         probeHashGenerator = other.probeHashGenerator;
+        sharedPartitionedMemoryContext = other.sharedPartitionedMemoryContext;
```

Comment:
Not sure what are the semantics of `LookupJoinOperatorFactory.duplicate`, but are you sure passing *same* `sharedPartitionedMemoryContext` should be here?

I looked into usages but I didn't raise confidence about this.